owncloudComLibrary/src/test/java/com/owncloud/android/lib/CookieJarImplTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.owncloud.android.lib
+
+import com.owncloud.android.lib.common.http.CookieJarImpl
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertFalse
+import junit.framework.Assert.assertTrue
+import okhttp3.Cookie
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import org.junit.Test
+
+class CookieJarImplTest {
+
+    private val oldCookies = ArrayList<Cookie>().apply {
+        add(COOKIE_A)
+        add(COOKIE_B_OLD)
+    }
+
+    private val newCookies = ArrayList<Cookie>().apply {
+        add(COOKIE_B_NEW)
+    }
+
+    private val updatedCookies = ArrayList<Cookie>().apply {
+        add(COOKIE_A)
+        add(COOKIE_B_NEW)
+    }
+
+    private val cookieStore = HashMap<String, List<Cookie>>().apply {
+        put(SOME_HOST, oldCookies)
+    }
+
+    private val cookieJarImpl = CookieJarImpl(cookieStore)
+
+    @Test
+    fun testContainsCookieWithNameReturnsTrue() {
```

Comment:
```suggestion
    fun `test contains cookie with name - ok`() {
```

Maybe we can use this style of function here to keep consistent with all the other tests.

Reply:
Great, I will update lists initialization too.